src/os/FileJournal.cc

```diff
@@ -1906,6 +1963,7 @@ void FileJournal::get_header(
   off64_t next_pos = pos;
   bufferlist bl;
   uint64_t seq = 0;
+  dout(2) << __FUNCTION__ << dendl;
```

Comment:
Minor: maybe we try use __FUNC__ instead of __FUNCTION__ since __FUNC__ is a part of C99 and __FUNCTION__ is kept to try to make compatible backend.